projects/plugins/jetpack/extensions/blocks/sharing-button/edit.js

```diff
@@ -21,10 +21,17 @@ const SharingButtonEdit = ( { attributes, context } ) => {
 		className: 'jetpack-sharing-button__list-item',
 	} );
 
+	const defaultIconColor = styleType === 'icon' ? '#fff' : '#000';
```

Comment:
Nitpicking here, but I'm wondering if we can force the white color for icons in CSS rather than here.

For other block styles, I'd say maybe let's inherit the body color instead of using pure black. Discard my comment if that was already discussed.

Reply:
Thanks for the suggestion. I was too focused on making it work and didn't think of the obvious.